### ISSUE TITLE
Consider using moneyphp

### ISSUE THREAD

**Issue by mblarsen** [indiehd/web-api#163]
https://github.com/moneyphp/money

Reimplementing code that deals with money, fractions, discounts, currencies is tiresome and error prone. 

You could cast directly to a money instance using [custom casts](https://laravel.com/docs/7.x/eloquent-mutators#custom-casts) much the same way dates works with Carbon

One such place would be:

```php
            $table->unsignedDecimal('full_album_price', 8, 4)->nullable();
```

**Comment by cbj4074** [comment]
I couldn't agree more! Thanks for bringing this library to our attention; it sounds perfect.

**Comment by mblarsen** [comment]
I found it while looking at Laravel Cachier which uses it too.

Should I try to use that in the casts PR #164? If tests are written it shouldn't break anything/too much.

So the money values are cast directly to Money.

**Comment by cbj4074** [comment]
Yes, absolutely. This seems like a good opportunity to integrate it.

**Comment by mblarsen** [comment]
Is the API in USD only? Maybe we should stick it in a config?

**Comment by cbj4074** [comment]
I like the idea of making the currency unit configurable.

**Comment by mblarsen** [comment]
It is now a part of #164

**Comment by cbj4074** [comment]
Fixed in #164
